### PR TITLE
MET-0: Update metrics documentation for topk/bottomk latest

### PR DESCRIPTION
**Pull request by michaljmatusiak92**
## Purpose of this pull request

Spotted that the documentation does not reflect the current state of the metrics operator. Applied the fix.

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/metrics/metrics-operators/bottomk.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Where: 
 | count | Rank matching time series by the count of the metric values across the time range, and return the  bottom n time series.  |
 | sum | Rank matching time series by the sum of the metric values across the time range, and return the bottom n time series. |
 | pct(n) | Return the n time series for which the calculated percentile of the metric values across the time range was lowest, and return the bottom n. |
-| latest | Rank matching time series by when the most recent data point was received, and return the bottom n time series.  |
+| latest | Rank matching time series by the value of the most recently ingested datapoint, and return the bottom n time series.  |
```
Comment (pkarwacki) [inline docs/metrics/metrics-operators/bottomk.md:30]: datapoint -> data point (same below)

**Review by pkarwacki — APPROVED** [review]

**Comment by jpipkin1** [comment]
Created PR #2928 to add @michaljmatusiak92 to the .clabot file.

File: docs/metrics/metrics-operators/bottomk.md (hunk 1 above)
Comment (michaljmatusiak92) [reply docs/metrics/metrics-operators/bottomk.md]: thanks, fixed

**Comment by jpipkin1** [comment]
@michaljmatusiak92, thank you for your submission! Make sure you click the "sign your name here" link in the **cla-bot** box above to get added to the approved list of contributors.

**Comment by michaljmatusiak92** [comment]
Done @jpipkin1 !

**Review by jpipkin1 — APPROVED** [review]
Approved, @michaljmatusiak92.

**Pull request merged**